awsapprunner/1.0.0/README.md

```diff
@@ -0,0 +1,82 @@
+# AWS App Runner Integration
+
+Keptn can integrate with AWS App Runner in 3 ways:
+
+1. Fire-and-forget triggering of App Runner from Keptn
+2. Fire-and-wait triggering of App Runner from Keptn
+3. Trigger Keptn from App Runner
+
+
+# Fire and Forget Trigger App Runner
+
+![](https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/aws-app-runner/assets/fire-and-forget-sequence.svg)
```

Comment:
This should be `https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/awsapprunner/1.0.0/assets/fire-and-forget-sequence.svg`

So `keptn-contrib` instead of `keptn-sandbox` and including the version number (1.0.0)